flink-yarn/src/main/java/org/apache/flink/yarn/YarnClusterClientV2.java

```diff
@@ -146,7 +146,9 @@ public ApplicationStatus getApplicationStatus() {
 
 	@Override
 	public void finalizeCluster() {
-		// Do nothing
```

Comment:
Please revert the changes here. This file is part of another big initiative (referred to as FLIP-6); we shouldn't touch it unless necessary. (basically, I don't know about the yarn-client life-cycle here so let's be conservative.)